docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -135,6 +135,75 @@ If you are using an older version, please upgrade.
 
         {% include /util/screenshot.html source="/images/brickman/wired-status-online-connect-automatically-selected.png" %}
 
+
+*   {: tab="Debian_commandline"}{% include /style/icon.html type="info" %}
```

Comment:
There doesn't seem to be anything Debian-specifc here. Maybe we could just call this "linux_command_line"?